modules/tensor_mechanics/doc/content/source/materials/ADRankTwoInvariant.md

```diff
@@ -1,20 +1,20 @@
-# Rank Two Invariant
+# ADRank Two Invariant
 
 !syntax description /Materials/ADRankTwoInvariant
 
 ## Description
 
-This is an ADMaterial model used to extract an invariant of a rank-2 tensor in a
+This is a ADMaterial model used to extract an invariant of a rank-2 tensor in a
```

Comment:
Umm, I'm pretty sure you use 'an' before a word that starts with an vowel.

I'll not hold up this PR for this, though.  Let's keep the AD and non-AD pages here for the time being, assuming the documentation builds fine, but we need to figure out a better long-term solution.

Reply:
Ha!  You are right.  I just added AD into the duplicate non-AD version.  It would make more sense to have the one md file that describes it's ability to do both things.